docs/release-notes/multirm-for-k8s.rst

```diff
@@ -8,7 +8,8 @@
    ``additional_resource_managers`` and ``resource_pools`` (additional resource managers are
    required to have at least one resource pool defined). Additional resource managers and their
    resource pools must have unique names. For more information, visit :ref:master configuration
-   <master-config-reference>.
+   <master-config-reference>. Support for notebooks and other workloads that require proxying is
+   being worked on and will be announced in a later release.
```

Comment:
Support for notebooks and other workloads that require proxying is currently under development.

Reply:
Alternatively, we could just leave it out.